XouDevSpec/Classes/features/ProfileScreen/ViewModel/ProfileViewModel.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  ProfileViewModel.swift
+//  XouDevSpec
+//
+//  Created by Xander Schoeman on 2020/04/27.
+//
+
+import Foundation
+
+public class ProfileViewModel {
+    
```

Comment:
Empty classes should be removed as it unused code.

Reply:
done and done